README.md

```diff
@@ -161,157 +215,60 @@ with tf.Session() as sess:
-```shell
-# 1. Install the wandb library
-pip install wandb
+# Tutorials
 
-# 2. Run a script that has the Trainer to automatically logs metrics, model topology and gradients
-python run_glue.py \
- --report_to wandb \
- --model_name_or_path bert-base-uncased \
- --task_name MRPC \
- --data_dir $GLUE_DIR/$TASK_NAME \
- --do_train \
- --evaluate_during_training \
- --max_seq_length 128 \
- --per_gpu_train_batch_size 32 \
- --learning_rate 2e-5 \
- --num_train_epochs 3 \
- --output_dir /tmp/$TASK_NAME/ \
- --overwrite_output_dir \
- --logging_steps 50
-```
-
-- **[Try in a colab →](http://wandb.me/hf)**
-- [Docs](https://docs.wandb.com/library/integrations/huggingface)
-
-# 🧹 Optimize hyperparameters with Sweeps
-Use Weights & Biases Sweeps to automate hyperparameter optimization and explore the space of possible models.
-
-### [Get started in 5 mins →](https://docs.wandb.com/sweeps/quickstart)
-### [Try Sweeps in PyTorch in a Colab →](http://wandb.me/sweeps-colab)
-
-### Benefits of using W&B Sweeps
-- **Quick to set up:** With just a few lines of code you can run W&B sweeps.
-- **Transparent:** We cite all the algorithms we're using, and our code is [open source](https://github.com/wandb/sweeps).
-- **Powerful:** Our sweeps are completely customizable and configurable. You can launch a sweep across dozens of machines, and it's just as easy as starting a sweep on your laptop.
-
-<img src="https://gblobscdn.gitbook.com/assets%2F-Lqya5RvLedGEWPhtkjU%2F-LyfPCyvV8By5YBltxfh%2F-LyfQsxswLC-6WKGgfGj%2Fcentral%20sweep%20server%203.png?alt=media&token=c81e4fe7-7ee4-48ea-a4cd-7b28113c6088" width="400" alt="Weights & Biases" />
-
-### Common use cases
-- **Explore:** Efficiently sample the space of hyperparameter combinations to discover promising regions and build an intuition about your model.
-- **Optimize:**  Use sweeps to find a set of hyperparameters with optimal performance.
-- **K-fold cross validation:** [Here's a brief code example](https://github.com/wandb/examples/tree/master/examples/wandb-sweeps/sweeps-cross-validation) of _k_-fold cross validation with W&B Sweeps.
+Explore example Colab Notebooks at [wandb/examples GitHub repository](https://github.com/wandb/examples/tree/master/colabs). Here are some of our favorites:
 
-### Visualize Sweeps results
-The hyperparameter importance plot surfaces which hyperparameters were the best predictors of, and highly correlated to desirable values for your metrics.
+[INSERT]
 
-<img src="https://paper-attachments.dropbox.com/s_194708415DEC35F74A7691FF6810D3B14703D1EFE1672ED29000BA98171242A5_1578695757573_image.png" width="720" alt="Weights & Biases" />
+&nbsp;
 
-Parallel coordinates plots map hyperparameter values to model metrics. They're useful for honing in on combinations of hyperparameters that led to the best model performance.
+# Try out our MLOps Course
 
-<img src="https://i.imgur.com/THYXBN0.png" width="720" alt="Weights & Biases" />
+Get MLOps Certified With The Course From Weights & Biases.
```

Comment:
Do we want to link to the MLOps course?

Reply:
@morganmcg1? I don't mind, it's an awesome course!